alternator/executor.cc

```diff
@@ -1854,11 +1857,42 @@ std::unordered_set<std::string> calculate_attrs_to_get(const rjson::value& req,
     return {};
 }
 
-static std::optional<rjson::value> describe_single_item(schema_ptr schema,
+void executor::describe_single_item(const cql3::selection::selection& selection,
+    const std::vector<bytes_opt>& result_row,
+    const std::unordered_set<std::string>& attrs_to_get,
+    rjson::value& item,
+    bool include_all_embedded_attributes) 
```

Comment:
Because I don't need/want the full monty of the wrapping function, and `include_all_embedded_attributes` is because we literally want everything in `executor::ATTRS_COLUMN_NAME`. Without having to figure the key names out. See comment in GetRecords on why we don't want `attrs_to_get` empty.